file_format.md

```diff
@@ -0,0 +1,193 @@
+# .simularium JSON Format
+
+JSON files accepted by the simularium-viewer contain the following data in JSON format:
+* **trajectory info**
+  * timeStepSize - the amount of time that passes in the simulation for each time step
```

Comment:
version is missing.  This doc is documenting version 1 and should make that clear.

Reply:
whoops